showmehow-files/fortune

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/app/games/fortune $@
```

Comment:
(Note: gets installed by flatpak. Nut maybe it should be installed by the makefile...)

Reply:
Now that I think of it this should be installed by flatpak - it is part of the cowsay/fortune dependency and should be installed into the tree by their manifests